tests/eos_bashrc

```diff
@@ -0,0 +1,152 @@
+# Note: you need to configure your eosroot according to your folder structure
```

Comment:
Having a bashrc here seems odd. I don't think this is a good way to maintain a multi-developer project.  Please create a `tests/script` directory and create individual scripts in there instead of having one large bashrc file.